packages/did/src/Did.utils.ts

```diff
@@ -494,5 +527,15 @@ export async function upgradeDid(
     }
   }
 
-  return writeDidFromPublicKeys(signer, submitter, newDidPublicKeys)
+  return writeDidFromPublicKeysAndServices(
+    signer,
+    submitter,
+    newDidPublicKeys,
+    lightDid.getEndpoints().map((service) => {
```

Comment:
Can we remove `lightDid.getEndpoints()` from the `return` for readability?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/428/commits/4adad47f6e2b3a8063cd7aca7da17b78421a9f24